js/src/personal/kelly/Kelly.page.tsx

```diff
@@ -1,34 +1,12 @@
-import { Title, Text } from '@mantine/core'
+import { Title } from '@mantine/core'
 import styles from './Kelly.module.css'
 
 export function KellyPage() {
   return (
     <>
-      <Title
-        className={styles.title}
-        ta="center"
-        mt={100}
-      >
-        Welcome to{' '}
-        <Text
-          inherit
-          variant="gradient"
-          component="span"
-          gradient={{ from: 'green', to: 'blue' }}
-        >
-          Kelly's Page
-        </Text>
-      </Title>
-      <Text
-        c="dimmed"
-        ta="center"
-        size="lg"
-        maw={580}
-        mx="auto"
-        mt="xl"
-      >
-        Hey! I'm Kelly.
-      </Text>
+      <div className="title_container">
```

Comment:
Does this actually apply?
It should likely be `className={styles.titleContainer}` since it's from the module.

Reply:
It does!

<img width="1461" alt="image" src="https://github.com/arklian/patina/assets/76794780/2229e6b9-4922-4b51-aa36-706e4ece0224">

Reply:
Hmmm this might mean that the modules aren't correctly adding hashes to the style names.
I may need to look at how it is.

You should likely still use styles.titleContainer.

Also if you want me to actually read your reply, and it's not just like `Done`, then leave the conversation as unresolved.